glider_dac/models/deployment.py

```diff
@@ -175,6 +176,22 @@ def on_complete(self):
                     if f.endswith(".md5"):
                         os.unlink(os.path.join(dirpath, f))
 
+    def calculate_checksum(self):
+        '''
+        Calculates a checksum for the deployment based on the MongoKit to_json()
+        serialization and the modified time(s) of the dive file(s).
+        '''
+        md5 = hashlib.md5()
+        # First ad the Mongo to_json serlialization
+        md5.update(self.to_json())
+        # Now add the modified times for the dive files in the deployment directory
+        # We dont MD5 every dive file here to save time
+        for dirpath, dirnames, filenames in os.walk(self.full_path):
+            for f in filenames:
+                if f in ["deployment.json", "wmoid.txt", "completed.txt"] or f.endswith(".md5"):
```

Comment:
We no longer use or rely on deployment.json, completed.txt or wmoid.txt because it required maintaining synchronization between the database and the filesystem which always seemed to be out of sync which caused processing errors.